Formula/p/pikchr.rb

```diff
@@ -0,0 +1,20 @@
+class Pikchr < Formula
+  desc "PIC-like markup language for diagrams in technical documentation"
+  homepage "https://pikchr.org/"
+  url "https://github.com/drhsqlite/pikchr/archive/refs/tags/version-1.0.0.tar.gz"
+  sha256 "55b469fbcc0ebc1b72483760a73a9ba90491fb3b03ac56fefeb71316f8007aae"
+  license "0BSD"
+
+  def install
+    system "make", "CC=#{ENV.cc}"
```

Comment:
Does it not pick up CC otherwise?

Reply:
The makefile uses gcc, which works on my machine, but I thought using the env value would be more appropriate

Reply:
We can use this, but it might also be good to submit a change upstream so that it does not force GCC.

Reply:
On my Mac, /usr/bin/gcc is a hard link to clang, so I could remove the "CC=#{ENV.cc)" and it would work exactly the same

Reply:
Hi could we please move this forward?

Reply:
Yeah, but my review remains unchanged. It would be good to submit a change upstream not to force GCC.

Reply:
I don't know how it should look like.  Are you recommending to drop `CC=gcc` in the Makefile?  Or to do something else?

Reply:
There are two options really:
- Make a pull request to the GitHub repository for pikchr so that it correctly builds on macOS without setting this (or hardlinks from expected GCC paths).
- Create an issue in the GitHub repository explaining that you want to build this for a system where `/usr/bin/gcc` is not what you want to use as a compiler (for example people using Linuxbrew on CentOS 7). And ask if upstream can make it configurable.

Reply:
I removed the `, "CC=#{ENV.cc}"` and it builds successfully

Reply:
I don’t think it makes sense for upstream to add more configurability to the makefile. It works correctly out of the box on my Mac, so I don’t have any evidence that it is needed. 

On the other hand, if you want to configure which compiler to use, you can easily do it by overriding the CC variable on the command line. It’s certainly due to me not understanding all the issues, but I don’t see what is the problem here.

If I’m to submit a patch upstream I need a more solid argument than “we need to be able to change compiler“, because this is already possible